libs/seiscomp/core/recordsequence.h

```diff
@@ -175,6 +175,8 @@ class SC_SYSTEM_CORE_API TimeWindowBuffer : public RecordSequence {
 		//! Return percentage of available data within TimeWindowBuffer's TimeWindow
 		double availability() const;
 
+		//! Return the buffered TimeWindow
```

Comment:
To be pedantic, this is actually not the buffered time window. The buffered time window is returned with `RecordSequence::timeWindow`. I am also not sure about a better name, maybe `capacity`. But note that the actual buffered time window can be larger than the requested time window because records may overlap the start and/or end time.

Reply:
"To be pedantic", you have to be, the API needs to be clear, I understand that. Maybe it was the naming issue that prevented the implementation from providing a method for that ;) What about `sequenceTimeWindow`?

Reply:
It is really difficult. Actually the RingBuffer suffers as well from correct naming. When I think about `numberOfRecords` one might expect the number of records currently held by the sequence. What is required is a naming prefix that gives access to the configuration and is consistent with both sequence implementations. Just a proposal:

```
configTimeWindow()
configNumberOfRecords()
configTimeSpan()
```

I am not sure if I like that ;) For the number of records `maxRecords` is probably the best but that does not work for the other attributes because of the overlap. Otherwise someone could complain that `maxTimeWindow() < timeWindow()`. That does not follow your expectation.

Reply:
I agree. Let's think about the naming  and then I will be back with a proposal

Reply:
And here is the proposal. What about: 
```
timeWindowToStore()
numberOfRecordsToStore()
timeSpanToStore()

```

Reply:
I can't come up with a better suggestion so please go ahead.

Reply:
Thanks, I updated the PR